Resources/Locale/en-US/holopad/holopad.ftl

```diff
@@ -141,6 +147,8 @@ holopad-service-musician = Service - Musician
 holopad-service-mime = Service - Mime
 holopad-service-clown-mime = Service - Clown/Mime
 holopad-service-gameroom = Service - Game Room
+holopad-service-breakroom = Service - Breakroom
```

Comment:
isnt this just General - Lounge?

Reply:
Perhaps. It's possible I've not really heard them referred to as lounges before. If it's redundant, I can remove it.

Reply:
What I mean is those rooms with a table and some chairs that are usually around dorms. Or at least thats how ive been mapping it.